ap/tasks/selection.py

```diff
@@ -46,25 +46,32 @@ def requires(self):
         return reqs
 
     def output(self):
-        return {
+        outputs = {
             "res": self.local_target(f"results_{self.branch}.parquet"),
             "stat": self.local_target(f"stats_{self.branch}.json"),
         }
 
+        # add additional columns in case the selector produces some
+        if self.selector_func.produced_columns:
+            outputs["cols"] = self.local_target(f"columns_{self.branch}.parquet")
```

Comment:
See 'descriptive power of keys' above
```suggestion
            outputs["columns"] = self.local_target(f"columns_{self.branch}.parquet")
```

Reply:
Yep, added that ✓